03.bowling/bowling.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << s
+end
+
+point = 0
+frames.each_with_index do |frame, idx|
```

Comment:
```suggestion
frames.each_with_index(1) do |frame, idx|
```

とすればもう少しコードがシンプルになるかも？